DEV branch won't build with Intel compiler

I got the following compile error when build with ICC:

OpenSubdiv/regression/far_regression/../../regression/common/cmp_utils.h(58): error: a template argument may not reference a local type
          std::vector<LevelMap> maps;
                      ^
          detected during instantiation of "void GetReorderedHbrVertexData(const OpenSubdiv::v3_0_0_beta::Far::TopologyRefiner &, const OpenSubdiv::v3_0_0_beta::HbrMesh<T> &, std::vector<T, std::allocator<T>> *, std::vector<bool, std::allocator<bool>> *) [with T=xyzVV]" at line 161 of "/usr/pic1/OpenSubdiv/regression/far_regression/far_regression.cpp"

compilation aborted for /usr/pic1/OpenSubdiv/regression/far_regression/far_regression.cpp (code 2)
make[2]: *** [regression/far_regression/CMakeFiles/far_regression.dir/far_regression.cpp.o] Error 2
make[1]: *** [regression/far_regression/CMakeFiles/far_regression.dir/all] Error 2
make: *** [all] Error 2

I need to turn on c++11 option for ICC to make it compile. I add the following line in CMakeLists.txt line 200:

list(APPEND OSD_COMPILER_FLAGS -std=c++11)        

Please take a look.

--- a/regression/common/cmp_utils.h
+++ b/regression/common/cmp_utils.h
@@ -31,6 +31,15 @@
 
 //------------------------------------------------------------------------------
 
+namespace {
+    template <class Face, class Edge, class Vertex>
+    struct LevelMapT {
+        std::vector<Face *>     faces;
+        std::vector<Edge *>     edges;
+        std::vector<Vertex *>   verts;
+    };
+};
+
 
 // Copies vertex data from hmesh into hbrVertexData reordered to match
 // the given refiner and subdivision level.  This is used for later easy 
@@ -48,13 +57,7 @@ GetReorderedHbrVertexData(
     typedef OpenSubdiv::HbrHalfedge<T> Hhalfedge;
 
     struct Mapper {
-
-        struct LevelMap {
-            std::vector<Hface *>     faces;
-            std::vector<Hhalfedge *> edges;
-            std::vector<Hvertex *>   verts;
-        };
-
+        typedef LevelMapT<Hface, Hhalfedge, Hvertex> LevelMap;
         std::vector<LevelMap> maps;
 
         Mapper(const OpenSubdiv::Far::TopologyRefiner &refiner, 

